src/docs/development/data-and-backend/google-apis.md

```diff
@@ -138,12 +138,11 @@ Use the API to create the desired API type and call methods, for instance:
 
 <?code-excerpt "lib/main.dart" skip="125" take="6"?>
 ```dart
-var youTubeApi = YouTubeApi(httpClient);
```

Comment:
I suspect this also is incorrect. 

FYI @kevmoo

Reply:
Same here, added a doc region.